docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.9"
+
+services:
+  backend:
+    build:
+      context: ./server
+      dockerfile: Dockerfile
+    container_name: my-backend
+    ports:
+      - "5000:5000"
```

Comment:
Ports on the back end need to be removed. Traffic should be go inside docker-compose network